janis_bioinformatics/tools/dawson/workflows/variantcalling/multisample/freebayes/freebayessomaticworkflow.py

```diff
@@ -190,15 +190,15 @@ def constructor(self):
         )
 
         self.step(
-            "allelicPrimitves",
+            "allelicPrimitives",
```

Comment:
I think it should be fine to rename this step here? Tried to replace all its occurrences.